### ISSUE TITLE
Column type JSON unhandled

### ISSUE THREAD

**Issue by bagage** [go-spatial/tegola#664]
When you try to use a column of type JSON, tegola fails:

> util.go:155: 114 type is not supported. (Expected it to be a stringer type)
> map.go:202: err fetching tile (z: 7, x: 64, y: 46) features: for layer (cities-point) unable to convert field [3] (string) of type (114 - json) to a suitable value: map[dates:map[2017:3044 2018:4] simplified:[]]

It would be awesome if it could be supported :).

**Comment by gdey** [comment]
Hi bagage,

Thanks for submitting the issue. 

What are you trying to do with the JSON value? Is it for the tags? What are you expecting to happen with the JSON value?

Thank you,

Gautam.

**Comment by bagage** [comment]
I'm just trying to select it via `select myTable.jsonField`, so that it is available through `myGeoson.properties.jsonField` after. For now, I've "fixed" the issue by forcing the cast `cast(myTable.jsonField as text) as jsonField` but it's a bit ugly, I believe JSON type could be stringified directly in tegola?

**Comment by ARolek** [comment]
@bagage we should be able to stringify the JSON type in tegola. Just to make sure I follow what you're after, are you looking to have the entire JSON structure as a MVT tag, or are you hoping to extract specific values from the JSON via the tegola config?

**Comment by bagage** [comment]
I am not sure about MVT meaning but I do not want to filter the json column in tegola config, but to return it completely :)

**Comment by ARolek** [comment]
@bagage yep, that's doable. So you plan on parsing the JSON client-side and then pulling out the values you want to present the user?

**Comment by bagage** [comment]
Yes exactly!
